README.md

```diff
@@ -23,6 +23,16 @@
 
 ## Installation
 
+### Using pip
+
+```bash
+pip install https://github.com/IntelLabs/MART/archive/refs/tags/<VERSION>.zip
+```
+
+Replace `<VERSION>` with the MART's version you want to install.
+
```

Comment:
Can we add an example of `pip install https://github.com/IntelLabs/MART/archive/refs/tags/v0.2.1.zip`?